dapp/src/hooks/useSwapEstimator.js

```diff
@@ -882,6 +892,7 @@ const useSwapEstimator = ({
         canDoSwap: true,
         gasUsed: gasEstimate,
         // TODO: should this be rather done with BigNumbers instead?
```

Comment:
Should this comment still apply specifically to `amountReceived` or does it apply to `inputAmount` as well now?

Reply:
I am not sure about that comment. I don't think it should apply to `inputAmount`.